src/styles/utils/_mixin.scss

```
@@ -0,0 +1,4 @@
+@mixin logo ($size) {
```

міксін лого повинен добавити логатип, а цей робить квадрат

```suggestion
@mixin square ($size) {
```